src/doc/rustdoc/src/write-documentation/linking-to-items-by-name.md

```diff
@@ -103,6 +103,27 @@ macro_rules! foo {
 }
 ```
 
+The following suffixes are available:
```

Comment:
I wouldn't teach this as a suffix, just when teaching macros mention that the `!` can be followed by an optional `()`, `{}`, or `[]`.